TAB2MXL/src/main/java/TAB2MXL/Measure.java

```diff
@@ -6,43 +6,42 @@
 	
 	public int measureNumber;
 	
-	public ArrayList<Note> notelist;
+	public ArrayList<Note> noteList;
 	public static int divisions;
 		
-	public int keyFifths=0;//always 0
+	public int keyFifths = 0; //always 0
 	
-	public int timeBeats;// user input 	
-	public int timeBeatType;// user input
+	public int timeBeats; // user input 	
+	public int timeBeatType; // user input
 	
-	public static String clefSign = "TAB";
-	public int clefLine;
+	public static final String clefSign = "TAB"; //"TAB" for Guitar
+	public int clefLine = 5; //"5" for Guitar
+
 
 	// Create a new measure
 	public Measure(int clefLine, int timeBeats, int timeBeatType, int measureNumber) {
 		
 		this.measureNumber=measureNumber;
-		
-		this.notelist = new ArrayList<Note>();
-		
+		this.noteList = new ArrayList<Note>();
 		Measure.divisions = 2; // Change this?
-		
-		this.clefLine=clefLine;		
```

Comment:
Not sure why we declare `clefLine = 5` as a global variable if we need it as an input to the constructor